kata-deploy/kata-cleanup.yaml

```diff
@@ -0,0 +1,80 @@
+---
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: kubelet-cri-o-kata-cleanup
+  namespace: kube-system
+spec:
+  selector:
+      matchLabels:
+        name: kubelet-cri-o-kata-cleanup
+  template:
+    metadata:
+        labels:
+          name: kubelet-cri-o-kata-cleanup
+    spec:
+      serviceAccountName: kata-label-node
+      nodeSelector:
+          container-runtime: cri-o
+          kata-runtime: cleanup
+      containers:
+      - name: kube-kata-cleanup
+        image: egernst/kata-deploy
+        imagePullPolicy: Always
+        command: [ "sh", "-c" ]
+        args:
+        - kubectl label node $NODE_NAME container-runtime- kata-runtime-;
+          systemctl daemon-reload && systemctl restart crio && systemctl restart kubelet;
+          while true;
+          do sleep 36000; done;
+        env:
+        - name: NODE_NAME
+          valueFrom:
+            fieldRef:
+              fieldPath: spec.nodeName
+        securityContext:
+          privileged: false
+  updateStrategy:
+    rollingUpdate:
+      maxUnavailable: 1
+    type: RollingUpdate
+---
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: kubelet-containerd-kata-cleanup
+  namespace: kube-system
```

Comment:
can we combine this into the above daemonset?
```
      nodeSelector:
          kata-runtime: cleanup
```
and 
```
systemctl daemon-reload; systemctl restart crio;  systemctl restart containerd; systemctl restart kubelet;
```

Reply:
agreed -done